users-service/service.yml

```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
 
 messages:
   receives:
-    - users.create
-    - users.list
-    - user.get-details
-    - user.update
-    - user.delete
+    - create users
+    - list users
+    - get users details
```

Comment:
`get user details`, I think users service only lists the details of one user at a time.